servicebus/data-plane/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Actual Behavior
```

Comment:
We don't need these .github files... it seems to overlap the ones we have at the root.

Reply:
Will update the changes in one of next PR.